using-timescaledb/reading-data.md

```diff
@@ -203,7 +203,7 @@ This query will output data in the following form:
  garage     | 10080 | {0,2679,957,2420,2150,1874,0}
 ```
 
-### Gap Filling [](gap-filling)
+### Gap Filling :timescale_function: :community_function: [](gap-filling)
```

Comment:
Aren't all of the functions on the API page timescale functions?

Reply:
This isn't the API page, this is the using timescale page.  In a bunch of the other examples, we flag when something is native postgres or timescale, so I'm just making consistent here.  Given that the other functions were things in OSS (e.g., `time_bucket`), I thought it was also worth flagging when in community/enterprise.